membership-attribute-service/app/models/Attributes.scala

```diff
@@ -59,6 +74,7 @@ case class DynamoAttributes(
   RecurringContributionPaymentPlan: Option[String] = None,
   MembershipJoinDate: Option[LocalDate] = None,
   DigitalSubscriptionExpiryDate: Option[LocalDate] = None,
+  PaperSubscriptionExpiryDate: Option[LocalDate] = None,
```

Comment:
we never actually write this field into dynamo! so cached responses to paper subscribers cause support asks to appear
https://github.com/guardian/members-data-api/blob/9970f2e01cd60d21abb0b77c7207b70785882746/membership-attribute-service/app/services/ScanamoAttributeService.scala#L58-L62